assets/shells/config.bat

```diff
@@ -9,14 +9,17 @@ set {{$K}}={{$V}}
 {{- end}}
 {{- end}}
 
-{{range $K, $CMD := .Scripts}}
-DOSKEY {{$K}}={{.Exec}} run "{{$CMD}}" $*
-{{end}}
+{{$execCmd := .ExecName}}
```

Comment:
I don't understand the point of this variable, It's never used as .ExecName, it will always be set to .ExecAlias on line 15? Seems you can just drop this and use .ExecAlias.

Reply:
The cmd string used should be "state" unless an alias is used. In that case, the absolute path is used.